.github/workflows/static-analysis.yml

```
@@ -0,0 +1,48 @@
+name: Static Analysis
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)